notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<ol><li><code>vae encoder and decoder</code> -&gt; <code>vae_encoder</code> and <code>vae_decoder</code></li><li>"The weight compression will be applied to footprint reduction." -&gt; "<em>Only</em> weight compression will be applied <em>for</em> footprint reduction."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>